app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<% if @items_to_buy != 0 %>
+     <h2 class="mb-4">Shopping Cart</h2>
+     <div class="d-flex justify-content-between align-items-start py-4">
+        <h4>Amount of food items to buy: <%= @recipe_foods.count %></h4>
+        <h4>Total value of food needed: $<%= @needed_money %></h4>
+     </div>
+
+    <table class="table table-bordered table-striped table-hover">
+        <thead>
```

Comment:
### Shopping List

Well done with the shopping list. The UI matches the wireframes  💯 

![image](https://github.com/olanikegloria/Recipe_app/assets/38088818/0c7f755c-653a-4741-8ea3-4c506d663a2c)

**[REQUIRED]**
The shopping list is a little bit tricky, in your case it does not quite meet the project requirements. This is because the shopping list should show the list of food that is **missing** for **all recipes of the logged-in user**. This means that you should compare the list of food for all recipes with the general food list of that user. 

- This is why the food list should be personal for each user, this will help track the food items inventory (quantity) of the logged-in user.

![image](https://github.com/olanikegloria/Recipe_app/assets/38088818/22387782-22ce-428f-b7ee-5b406c85fd36)

***Think hat*** 🪄 - Think of the shopping list as way to determine how many food items are needed for all recipes that you do not currently have on your food items.

Remember in one on the comments above I mentioned how the food items are like a list of ingredients you currently have at home?

**Scenario:** Say you have 2 tomatoes at home that you added to the food list (as `quantity: 2`). You create 3 recipes that have tomatoes added on the ingredients list. Say one recipe needs 5 tomatoes and the other two recipes need 1 tomato each. In total all the recipes need 7 tomatoes combined. You only have 2 tomatoes on your inventory therefore you need to buy 5 more tomatoes in order to have all ingredients you need for all recipes. Thus, the shopping list should have 5 tomatoes.

Essentially the shopping list is meant to help you shop for ingredients that you do not have but need for your recipes. The **SweetRecipe** app is like a planning tool to help you make the recipes you own at home.